mkdocs.yaml

```diff
@@ -33,5 +33,6 @@ nav:
       - CL Relations: relations_guide.md
       - How to resolve merge conflicts: resolving_merge_conflicts.md
       - Fixing unintended ^^xsd:string diffs: Fixing_xsdstring_diffs.md
+      - Taxon Constraints: taxon-restrictions.md
```

Comment:
Great.
May consider implementing consistent case types in bullets text:

E.g.,  
"Your ODK Repository Overview:" (title case) 
vs
"How to resolve merge conflicts:" (sentence case);

"Imports management:" (sentence case)
vs
"Taxon Constraints" (title case)

Reply:
will fix CL internal docs - the other i *think* needs to be fixed in ODK (think those are shipped with ODK, not sure if they update in the yaml file but I would prefer fixing it there, updating repo with next ODK release, change what needs to be changed